build.gradle.kts

```diff
@@ -320,12 +320,14 @@ tasks {
                    |   # For frontal shields, unfold time and projectile travel time are considered to determine flanking
                    |   # For modes that want to hit shields, reducing the threshold makes them more likely to fire
                    |   # For modes that want to avoid shields, the opposite is true
-                   |   ,"targetShields_threshold" : 0.1
-                   |   ,"avoidShields_threshold" : 0.3
+                   |   
+                   |   ,"targetShields_threshold" : 0.1     # i.e. Attack if target flux < 90% (simplified, see above)
```

Comment:
=)

Reply:
Threw me through a bit of a loop when trying to think about this initially, so I figure this comment might help out other people like me reading this